### PR TITLE
Dealing with EXDATES in ITip\Broker & fix for #126

### PR DESCRIPTION
**Pull request by armin-hackmann**
This PR solves two issues:
* parseEventInfo() only returned the value of the first `EXDATE` present in an object
* if an application swapped `EXDATE`-values in a PUT, the `$significantChangeHash` would imply a real change of the event happened

Also see the initial [issue](https://github.com/fruux/sabre-vobject/issues/126) & the thread in [this commit](https://github.com/armin-hackmann/sabre-vobject/commit/3a77fb067999c28dbb1d0c44d982f791005ea59a).

### CODE REVIEW

File: lib/ITip/Broker.php [hunk 1]
```diff
@@ -855,7 +855,10 @@ protected function parseEventInfo(VCalendar $calendar = null) {
                 $sequence = $vevent->SEQUENCE->getValue();
             }
             if (isset($vevent->EXDATE)) {
-                $exdate = $vevent->EXDATE->getParts();
+                foreach ($vevent->select('EXDATE') as $val) {
+                    $exdate = array_merge($exdate, $val->getParts());
```
Comment (Hywan) [inline lib/ITip/Broker.php]: Do we consider the `PRIORITY` property here to sort `EXDATE` before merging them? Because order is important when merging.

Comment (armin-hackmann) [inline lib/ITip/Broker.php]: As discussed via Slack yesterday you: there is no relation between PRIORITY and EXDATE. And the exdates get sorted (so we have earliest to latest in our variable) after merging values of all EXDATE-properties.

Did I miss something?

Comment (evert) [inline lib/ITip/Broker.php]: No, you are good :+1:

**Comment by armin-hackmann** [comment]
anything missing for you, @evert? :)

**Comment by evert** [comment]
Sorry for the delay armin. Just noticed this on my todo list from a few days ago :S

**Pull request merged**